src/app/Providers/components/ProvidersTable/VMware/VMwareProvidersTable.tsx

```diff
@@ -63,6 +63,11 @@ const VMwareProvidersTable: React.FunctionComponent<IVMwareProvidersTableProps>
     isEqual: (a, b) => a.name === b.name,
   });
 
+  const setInventoryDownloadURL = () =>
```

Comment:
I think we can just make this a string variable instead of a function that is only called once. Also, we should use the string template syntax and avoid concatenating strings with +.

```ts
const inventoryDownloadUrl = `${
  VIRT_META.inventoryPayloadApi
}/api/v1/extract?providers=${selectedItems.map((provider) => provider.name).join()}`;
```

Reply:
I've been going back and forth between string template and concatenating strings on that one! I'll use the former then.
Re string variable, yeah re-render is done at every user action.

Reply:
Yeah, I try to never use + for string concat, but the reasons are a bit vague. Something about type coercion, maybe performance? https://softwareengineering.stackexchange.com/questions/90203/why-is-so-bad-for-concatenation

Reply:
Thanks for the link.